initial-promises/node-server/promises.cf

```diff
@@ -70,7 +70,7 @@ bundle common va
 
   vars:
     policy_server::
-      "bs" slist => { "startExecution", "check_disable_agent", "clean_red_button", "update", "set_red_button", "internal_security", "process_matching", "check_cf_processes", "check_uuid", "check_cron_daemon", "garbage_collection", "check_binaries_freshness"
+      "bs" slist => { "startExecution", "check_disable_agent", "clean_red_button", "update", "set_red_button", "internal_security", "process_matching", "check_cf_processes", "check_cron_daemon", "garbage_collection", "check_binaries_freshness"
```

Comment:
We should keep the call to check_uuid and port the uuid checking code from techniques